packages/features/teams/hooks/useIsInTeam.ts

```diff
@@ -0,0 +1,10 @@
+import { trpc } from "@calcom/trpc/react";
+
+export const useIsInTeam = () => {
+  const { data, isLoading } = trpc.viewer.teams.list.useQuery(undefined, {});
+  const teams = data?.filter((team) => team.accepted) || [];
+  return {
+    isLoading,
+    isInTeam: teams.length > 0,
+  };
+};
```

Comment:
i like this. @CarinaWolli should we use this in other places where we check for teams?